pkg/autoscaler/statforwarder/forwarder_test.go

```diff
@@ -435,11 +435,9 @@ func TestProcess(t *testing.T) {
 	kubeClient.CoordinationV1().Leases(testNs).Create(ctx, anotherLease, metav1.CreateOptions{})
 	lease.Informer().GetIndexer().Add(anotherLease)
 
-	// Wait for the forwarder to become the leader for bucket1.
 	if err := wait.PollImmediate(10*time.Millisecond, 10*time.Second, func() (bool, error) {
-		p1 := f.getProcessor(bucket1)
-		p2 := f.getProcessor(bucket2)
-		return p1 != nil && p2 != nil && p1.holder == testHolder1 && p2.holder == testHolder2, nil
+		_, p1owned := f.getProcessor(bucket1).(*localProcessor)
+		return p1owned, nil
```

Comment:
The original test checked bucket2 as well.

Reply:
I added a check that p2 is remote